src/server/routes/calendar.js

```diff
@@ -68,11 +75,40 @@ router.post('/addevent', function(req,res){
             if(err){
               console.log(err);
             }
-            console.log("Event Inserted");
-          });
+            else{
+              console.log("Event Inserted Into Google Database");
+            }
+        });
       }
     }
   });
+
+  //Event Inserted into Kiddo Database
+  const newEvent = Event();
+
+  newEvent.title = eventJSON.title;
+  newEvent.startDateTime = eventJSON.startDate.concat(':00');
+  newEvent.endDateTime = eventJSON.endDate.concat(':00');
+  newEvent.calendarName = eventJSON.calendar;
+  newEvent.email = req.user.email;
+  
+  newEvent.save(function(err, data){
+    if(err){
+      console.log(err);
+    }
+    else{
+      console.log("Event Inserted into Kiddo DB");
+      //Insert Event ID into Users Table for User that Created Event
+      User.findOneAndUpdate({email: data.email}, {$push:{events:data._id}}, function(err, response){
+        if(err){
+          console.log(err);
+        }
+        else{
+          console.log("User Updated With New Event");
+        }
+      });
+    }
+  });
 });
 
-module.exports = router;
+module.exports = router;
```

Comment:
New line needed as POSIX standard demand
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline